cli/tests/integrations/test_marathon_groups.py

```diff
@@ -123,6 +123,56 @@ def test_update_missing_field():
             "Possible properties are: ")
 
 
+def test_scale_group():
+    _deploy_group('tests/data/marathon/groups/scale.json')
+    returncode, stdout, stderr = exec_command(['dcos', 'marathon', 'group',
+                                               'scale', 'scale-group', '2'])
+    assert stderr == b''
+    assert returncode == 0
+    watch_all_deployments()
+    returncode, stdout, stderr = exec_command(
+        ['dcos', 'marathon', 'group', 'show',
+         'scale-group'])
+    res = json.loads(stdout.decode('utf-8'))
+
+    assert res['groups'][0]['apps'][0]['instances'] == 2
+    _remove_group('scale-group')
+
+
+def test_scale_group_not_exist():
+    returncode, stdout, stderr = exec_command(['dcos', 'marathon', 'group',
+                                               'scale', 'scale-group', '2'])
+    assert stderr == b''
```

Comment:
Marathon rest API works is such way, when we want to scale  group which doesn't exist, end send PUT request, new group with empty applications list is created, and status OK returned.
 For example:
curl -X PUT -i http://rslyusarc-ElasticL-QY77PH81BNNO-1491641693.us-west-2.elb.amazonaws.com/marathon/v2/groups/not-exist-group -d '{"scaleBy": 2.5 }'
HTTP/1.1 200 OK
Cache-Control: no-cache, no-store, must-revalidate
Content-Type: application/json; qs=2
Date: Wed, 26 Aug 2015 08:23:30 GMT
Expires: 0
Pragma: no-cache
Server: openresty/1.7.10.2
X-Marathon-Leader: http://$($MESOS_IP_DISCOVERY_COMMAND):8080
Content-Length: 92
Connection: keep-alive

{"version":"2015-08-26T08:23:30.119Z","deploymentId":"2f0cf4af-2c69-4d8e-8dae-71eb589fb3f6"}

curl -X GET -i http://rslyusarc-ElasticL-QY77PH81BNNO-1491641693.us-west-2.elb.amazonaws.com/marathon/v2/groups/not-exist-group 
HTTP/1.1 200 OK
Cache-Control: no-cache, no-store, must-revalidate
Content-Type: application/json; qs=2
Date: Wed, 26 Aug 2015 08:27:35 GMT
Expires: 0
Pragma: no-cache
Server: openresty/1.7.10.2
X-Marathon-Leader: http://$($MESOS_IP_DISCOVERY_COMMAND):8080
Content-Length: 102
Connection: keep-alive

{"id":"/not-exist-group","apps":[],"groups":[],"dependencies":[],"version":"2015-08-26T08:23:30.119Z"}